DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/tableConfig.js

```diff
@@ -39,6 +44,31 @@ var employeeColumnDefinitions = [
   { responsivePriority: 1, targets: 0 },
   { responsivePriority: 2, targets: 1 },
   { responsivePriority: 3, targets: 2 },
+  {targets: 3, render: function(data) {
+      if(data === 31) {
+        return "Intellectual/Developmental Disability (IDD)"
+      }
+      if(data === 32) {
+        return "Psychiatric Disability (PD)";
+      }
+      if(data === 33) {
+        return "Visual Impairment (VI)";
+      }
+      if(data === 34) {
+        return "Hearing Impairment (HI)";
+      }
+      if(data === 35) {
+        return "Substance Abuse (SA)";
+      }
+      if(data === 36) {
+        return "Neuromuscular Disability (NM)";
+      }
+      if(data === 37) {
+        return "Age Related Disability (AR)";
```

Comment:
The linter should be picking up these double-quotes and flagging them but it is not... 🤷‍♂️